### PR TITLE
fix: update local-storage image tag

### PR DESCRIPTION
**Pull request by angel0507**
Signed-off-by: jie.li <jie.li@daocloud.io>

### CODE REVIEW

File: deploy/nginx-disk.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
     spec:
       restartPolicy: Always
       terminationGracePeriodSeconds: 0
-      schedulerName: localstorage-local-storage-scheduler
+      schedulerName: local-storage-scheduler
```
Comment (niulechuan) [inline deploy/nginx-disk.yaml:35]: According to @sun7927's advice we should add hwameistor prefix into all of hwameistor conponents.

CC: @sun7927 should we?

**Review by niulechuan — COMMENTED** [review]
some comments

File: deploy/nginx-disk.yaml (hunk 1 above)
Comment (sun7927) [reply deploy/nginx-disk.yaml:35]: schedulername should be hwameistor-scheduler which will work for both lvm and disk volume

Comment (angel0507) [reply deploy/nginx-disk.yaml]: ok, i had modify it by  replacing scheduler name

Comment (niulechuan) [reply deploy/nginx-disk.yaml]: > schedulername should be hwameistor-scheduler which will work for both lvm and disk volume

Got it

File: deploy/05_cluster.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 kind: DaemonSet
 apiVersion: apps/v1
 metadata:
-  name: localstorage-local-storage
+  name: hwameistor
```
Comment (sun7927) [inline deploy/05_cluster.yaml:4]: the daemonset name is suggested to be hwameistor-local-storage which means it is one of components of the hwameistore.

File: deploy/06_csi_controller.yaml [hunk 3]
```diff
@@ -1,17 +1,17 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: localstorage-local-storage-csi-controller
+  name: hwameistor-csi-controller
```
Comment (sun7927) [inline deploy/06_csi_controller.yaml:4]: this csi-controller is suggested to be renamed to hwameistor-local-storage-csi-controller. Later, local-disk-manager will deploy another csi-controller (e.g. hwameistor-local-disk-manager-csi-controller).

File: deploy/06_csi_controller.yaml [hunk 4]
```diff
@@ -22,8 +22,8 @@ spec:
               - key: app
                 operator: In
                 values:
-                - localstorage-local-storage
-            topologyKey: localstorage.hwameistor.io/local-storage-topology-node
+                - hwameistor
```
Comment (sun7927) [inline deploy/06_csi_controller.yaml:25]: should be hwameistor-local-storage?

File: deploy/nginx-lvm-ha.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  selector:
+    matchLabels:
+      app: nginx-local-storage-lvm-ha
+  template:
+    metadata:
+      labels:
+        app: nginx-local-storage-lvm-ha
+      name: nginx-local-storage-lvm-ha
+    spec:
+      schedulerName: local-storage-scheduler
```
Comment (sun7927) [inline deploy/nginx-lvm-ha.yaml:18]: schedulerName is not correct

File: deploy/storageclass-lvm-ha.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: local-storage-hdd-lvm-ha
+provisioner: localstorage.hwameistor.io
```
Comment (sun7927) [inline deploy/storageclass-lvm-ha.yaml:5]: provisioner is not correct. In addition, storageclass name had better to be in style like hwameistor-lvm-hdd-ha

File: deploy/05_cluster.yaml [hunk 7]
```diff
@@ -33,11 +33,11 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
-            - "--kubelet-registration-path=/var/lib/kubelet/plugins/localstorage.hwameistor.io/csi.sock"
+            - "--kubelet-registration-path=/var/lib/kubelet/plugins/lvm.hwameistor.io/csi.sock"
           lifecycle:
             preStop:
               exec:
-                command: ["/bin/sh", "-c", "rm -rf /registration/localstorage.hwameistor.io /registration/localstorage.hwameistor.io-reg.sock"]
+                command: ["/bin/sh", "-c", "rm -rf /registration/lvm.hwameistor.io /registration/lvm.hwameistor.io-reg.sock"]
```
Comment (niulechuan) [inline deploy/05_cluster.yaml:40]: Please notice the extra spaces behind ``/registration/lvm.hwameistor.io``

**Review by niulechuan — COMMENTED** [review]
some comments

**Pull request merged**